lib/openstack_query/queries/query_server.py

```diff
@@ -0,0 +1,124 @@
+from enums.query.server_properties import ServerProperties
+from enums.query.query_presets import (
+    QueryPresetsGeneric,
+    QueryPresetsDateTime,
+    QueryPresetsString,
+)
+
+from openstack_query.queries.query_wrapper import QueryWrapper
+from openstack_query.runners.server_runner import ServerRunner
+
+from openstack_query.handlers.client_side_handler_generic import (
+    ClientSideHandlerGeneric,
+)
+from openstack_query.handlers.client_side_handler_string import ClientSideHandlerString
+from openstack_query.handlers.client_side_handler_datetime import (
+    ClientSideHandlerDateTime,
+)
+
+from openstack_query.handlers.server_side_handler import ServerSideHandler
+from openstack_query.handlers.prop_handler import PropHandler
+
+from openstack_query.utils import convert_to_timestamp
+
+
+class QueryServer(QueryWrapper):
```

Comment:
Nit: I think we should be clearer that this is an adapter (following the adapter pattern) rather than executing the query directly.

I'm on the fence between something like
`ServerQueryAdapter`, `ServerQueryMapping` or something completely different? Thoughts?

Reply:
makes sense - I'll go with ServerQueryAdapter and rename the directory

Reply:
I don't want to rename this - simply because this will - most likely - be instantiated by users to run their queries if they wish to do so programmatically - naming it something like `ServerQueryAdapter` may confuse them

Reply:
Sure :+1: